src/UglyToad.PdfPig/Util/PdfImageHelper.cs

```diff
@@ -0,0 +1,160 @@
+﻿namespace UglyToad.PdfPig.Util
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using UglyToad.PdfPig.Content;
+    using UglyToad.PdfPig.Tokens;
+    using UglyToad.PdfPig.XObjects;
+
+    /// <summary>
+    /// Helper class for IPdfImages.
+    /// </summary>
+    public static class PdfImageHelper
+    {
+        const short TIFF_BIGENDIAN = 0x4d4d;
+        const short TIFF_LITTLEENDIAN = 0x4949;
+
+        const int IfdLength = 10;
+        const int HeaderLength = 10 + (IfdLength * 12 + 4);
+
+        /// <summary>
+        /// Parse the rawBytes of an XObjectImage to return a Tiff encoded byte[].
+        /// </summary>
+        public static bool TryGetTiff(this IPdfImage image, out byte[] bytes)
+        {
+            bytes = null;
+
+            if (image == null || image.GetType() != typeof(XObjectImage))
+            {
+                return false;
+            }
+            if(!image.TryGetBytes(out var lazyBytes))
+            {
+                lazyBytes = image.RawBytes;
+            }
+            var pureBytes = lazyBytes.ToArray();
```

Comment:
Here after removing the CcittFaxFilter implementation I think we want `var pureBytes = image.RawBytes;` since those are just the TIFF bytes anyway, but beforehand we want to check whether the `xImage.ImageDictionary` filters contains the CcittFax filter, I'll write up an implementation of this after I merge this PR.